Use static link for crypto and ssl libraries for rpmb_dev to fix libraries mismatch in host side

AjanZhong opened the pull request:
rpmb_dev is host tool when emulating RPMB feature, it depends on
crypto and ssl libraries on host side. Due to libraries version
mismatch on development machine and device under test, rpmb_dev
application might failed to launch.

Use static link to fix this issue.

sysopenci commented:
Improper Commit Message
signed off value not found in commit message

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4806

Done by "Joshi, Shweta" <shweta.joshi@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/2792

Done by "Joshi, Shweta" <shweta.joshi@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Joshi, Shweta" <shweta.joshi@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/3188

Commit:
Use static link for crypto and ssl libraries for rpmb_dev

rpmb_dev is host tool when emulating RPMB feature, it depends on
crypto and ssl libraries on host side. Due to libraries version
mismatch on development machine and device under test, rpmb_dev
application might failed to launch.

Use static link to fix this issue.

Tracked-On: OAM-105243
Signed-off-by: Zhong,Fangjian <fangjian.zhong@intel.com>
